src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -111,6 +111,10 @@ pub struct MachineConfig {
     /// Configures what page size Firecracker should use to back guest memory.
     #[serde(default)]
     pub huge_pages: HugePageConfig,
+    /// GDB socket address.
+    #[cfg(feature = "gdb")]
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub gdb_socket_address: Option<String>,
```

Comment:
I think we can call this `path` rather than `address` for consistency with other APIs. Also, we need to update https://github.com/firecracker-microvm/firecracker/blob/main/src/firecracker/swagger/firecracker.yaml to reflect this change.

Reply:
Much better thanks, I've updated